db/migrate/20221104104700_seed_initial_instance_id_setting_value.rb

```diff
@@ -0,0 +1,8 @@
+class SeedInitialInstanceIdSettingValue < ActiveRecord::Migration[6.0]
+  def up
+    setting = Setting.find_or_create_by(name: 'instance_id', category: 'Setting')
```

Comment:
I'm trying to wrap my head around why this needs to be a migration, vs a seed, but maybe @ezr-ondrej can enlighten us? :)

Reply:
I actually think a seed is better and we should recommend that to users.

Reply:
Updated to be a seed, but didn't actually test it myself (I am rather sure it just works, but you know)